providers/frame/utils/createCustomChainSpec.ts

```diff
@@ -0,0 +1,68 @@
+import { ss58, testUser } from "../../../crypto/mod.ts"
+
+const DEFAULT_TEST_USER_COUNT = 100_000
+const DEFAULT_TEST_USER_INITIAL_FUNDS = 1_000_000_000_000_000_000
```

Comment:
Validated that this is safe for `JSON.stringify`

Note: it's not valid for integer operations because it's greater than `Number.MAX_SAFE_INTEGER`
```
> 1000000000000000000 + 1 === 1000000000000000000 + 2
true
```